### DISCUSSION TITLE
Discrepancy regarding Hit Circles unfocus dim

### DISCUSSION THREAD

**Discussion by ItsShamed** [ppy/osu#23571] [Ideas] [answered]
I don't know if it's part of one of these differences that make lazer "not feel like stable", but at least that's one more documented.

It is already known that hit objects that are not within the hit window are slightly dimmed. Lazer seems to have the exact same dim amount as stable. __However__, while stable doesn't dim the hit circles' numbers, lazer does.

|  | Stable | Lazer |
|--------|--------|--------|
| Global screenshot | ![screenshot1113](https://github.com/ppy/osu/assets/39100084/593f4ca1-af14-44d2-9e53-cf2ba584d255) | ![osu_2023-05-16_20-57-57](https://github.com/ppy/osu/assets/39100084/45065db4-0002-456c-94b7-9e13355bbd79) |
| Close-up | ![image](https://github.com/ppy/osu/assets/39100084/4bbca077-4fad-465a-984e-3cffc09f5324) | ![image](https://github.com/ppy/osu/assets/39100084/e0cb9ac5-4737-45f2-97fb-a55fa11766a1) |

**Comment by peppy** [comment]
I was waiting for someone to notice this 😅.

Stable has a list of sprites/layers which are affected by dimming. I guess we'll need to respect this.

Of note, this simplification is mentioned in the implementation:

![JetBrains Rider 2023-05-17 at 05 20 08](https://github.com/ppy/osu/assets/191335/d1071054-af1e-41ef-aff9-db26965d79c1)

**Reply by peppy** [reply, marked as answer]
Tracking at https://github.com/ppy/osu/issues/24956.
